qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -0,0 +1,118 @@
+from tornado.gen import coroutine, Task
+
+from qiita_core.util import execute_as_transaction
+
+from .base_handlers import BaseHandler
+import redbiom.summarize
+import redbiom.search
+import redbiom._requests
+import redbiom.util
+import redbiom.fetch
+
+
+class RedbiomPublicSearch(BaseHandler):
+    @execute_as_transaction
+    def _get_context(self, callback):
+        df = redbiom.summarize.contexts()
+        result = df.ContextName.values
+        callback(result)
+
+    @coroutine
+    @execute_as_transaction
+    def get(self, search):
+        contexts = yield Task(self._get_context)
+        self.render('redbiom.html', contexts=contexts)
+
+    @execute_as_transaction
+    def _redbiom_search(self, context, query, search_on, callback):
+        df = redbiom.summarize.contexts()
+        contexts = df.ContextName.values
```

Comment:
per gchat, we may want to filter these. And one other possibility in addition to what we discussed was is to filter on load of the cache as well.